metabase/metabase
# Allow filtering on columns from all stages in dashboards

Closes #48253

### Description

The epic got changed because #47800 got merged in the meantime - we want dashboard filters to work the same as in filters modal in QB.

This PR:
- updates the stage indexes for which filters are available in dashboards
- updates tests after #47805
- updates tests after #47936
  - see https://github.com/metabase/metabase/issues/47184#issuecomment-2310056353

### How to verify

1. Create a multi-stage query (question or model) and add it to dashboard
2. It should be possible to map dashboard filters to every filter that is available in filter modal in query builder for a given entity

CI won't be green - 1 test is failing because of #48258
